tests/test_connector.py

```diff
@@ -504,6 +511,7 @@ def test_get_object_with_cookies(self):
             self.connector.get_object(objtype, {})
             self.assertEqual(None, self.connector.session.auth)
 
+
```

Comment:
extra line

Reply:
Pyflake mandates that 2 blank lines be used before a class declaration, so I think it makes sense here :)